runelite-client/src/main/java/net/runelite/client/eventbus/EventBus.java

```diff
@@ -72,7 +86,15 @@ private CompositeDisposable getCompositeDisposable(@NonNull Object object)
 			.cast(eventClass) // Cast it for easier usage
 			.take(takeUntil)
 			.doFinally(() -> unregister(lifecycle))
-			.subscribe(action, Sentry::capture);
+			.subscribe(action, error ->
+			{
+				log.error("Exception in eventbus", error);
+
+				if (RuneLiteProperties.getLauncherVersion() != null && OpenOSRSConfig.shareLogs())
+				{
+					Sentry.capture(error);
+				}
```

Comment:
Will this result in a double error being output?

Reply:
The logger isn't connected with Sentry

log.error is for the local log (client.log or TTY or whatever) and the other one is dispatched to Sentry